src/slack/constants.js

```diff
@@ -2,22 +2,74 @@ import * as vlConstants from './vacuumlabs/constants'
 import * as testConstants from './test/constants'
 import * as wincentConstants from './wincent/constants'
 
-export const MESSAGES = {
+const REASON = 'reason'
+const MANAGER = 'manager'
+
+const REASON_WITH_ADDRESS_QUESTION = {
+  [REASON]: ':question: Why do you need these items? Reply by sending a message. And don\'t forget to tell us your address!',
+}
+
+const REASON_DEFAULT_QUESTION = {
+  [REASON]: ':question: Why do you need these items? Reply by sending a message.',
+}
+
+const REASON_NOTE_QUESTION = {
+  [REASON]: ':pencil: Add a note by sending a message. And don\'t forget to tell us your address!',
+}
+
+const REASON_COMMENT_QUESTION = {
+  [REASON]: ':pencil: Send your comment in a message.',
+}
+
+const MANAGER_QUESTION = {[MANAGER]: ':question: Name of your manager (needed for approval for items above 100 EUR - write N/A otherwise). Reply by sending a message.'}
```

Comment:
I see you added the "Reply by sending a message." part. I get it, but suddenly the sentence before it doesn't make sense - it's not a sentence. I see two options:
- `:question: Name of your manager (needed for approval for items above 100 EUR - write N/A otherwise):`
  - the original that I sent you
  - by now the person should understand he should write a message, because they were asked for reason before
- `:question: What is the name of your manager? (Needed for approval for items above 100 EUR - write N/A otherwise.) Reply by sending a message.`
  - making the first sentence a question

I don't have a strong product opinion here, I think I'm leaning towards the second option. if you agree, please change it so

Reply:
Sure, I changed it to : 
`:question: Name of your manager (needed for approval for items above 100 EUR - write N/A otherwise):`